src/envoy/mixer/http_filter.cc

```diff
@@ -329,6 +329,7 @@ class ReportData : public HttpReportData {
     data->send_bytes = info_.bytesSent();
     data->duration =
         std::chrono::duration_cast<std::chrono::nanoseconds>(info_.duration());
+    data->response_code = 500;
```

Comment:
please add comment
// responseCode is for the backend response. If it is not valid, the request is rejected by Envoy
// set the response code for such requests as 500.

Reply:
Done. Thanks!